jetson/closed-loop/src/__main__.py

```diff
@@ -0,0 +1,79 @@
+import lcm
+from rover_msgs import ArmPosition
+
+
```

Comment:
what is this file

Reply:
Might've been written by someone on teleop but I don't think so. Committed by Josh Siegel who I think was signed in on rocker but I'm confused. Can be deleted as far as I'm concerned